doc/api/crypto.md

```diff
@@ -937,7 +938,7 @@ Example: Using the [`sign.update()`][] and [`sign.sign()`][] methods:
 
```

Comment:
It is better to add a description about a signature algorithm in the comment of examples like
```diff
-// Prints: the calculated signature
+// Prints: the calculated signature with an algorithm of the type of
+// private key with SHA256. When an RSA private key is used, the
+// signature algorithm is RSA-SHA256. It is ECDSA with SHA256 in case
+// of an EC private key.
```

Reply:
Write something to that effect.

Reply:
It looks good to me.